tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -1008,7 +1008,10 @@ <h2>Create a distance feature</h2>
                                            on-example-change="exampleChange_" id="viewer"
                                            display-mode="[[exampleDisplayMode]]"
                                            feature-search-value="{{featureSearchValue}}"
-                                           compare-title="[[compareTitle]]">
+                                           compare-title="[[compareTitle]]"
+                                           saliency="[[attribution]]"
+                                           show-saliency="true"
```

Comment:
you can omit `="true"` when specifying boolean prop.

Reply:
done